Yoast/wpseo-woocommerce
# Add option to display the price in Schema and OpenGraph with tax included

## Summary

This PR can be summarized in the following changelog entry:

* Adds an option to display the price in Schema and OpenGraph _with_ tax included.

## Relevant technical choices:

* Moved OpenGraph functionality to its own class and gave it a hook that all the specific fields hook on to, so people can unhook specific fields.
* Increased test coverage to 100% for the new `woocommerce-opengraph.php` class.
* Increased test coverage to 100% for the new `woocommerce-utils.php` class.
* Increased test coverage to 100% for the `woocommerce-schema.php` class.

## Test instructions

This PR can be tested by following these steps:

* Enable tax in your WooCommerce install (on the General tab of WooCommerce settings).
* Set the new setting in the admin to true:

![Screen Shot 2020-01-22 at 11 32 10](https://user-images.githubusercontent.com/487629/72888056-21d99c00-3d0d-11ea-8778-70abe7f16da9.png)

* Add a tax rate and play with the settings on the Tax tab, the outcomes should be the same on the frontend as they are in the schema.

Fixes https://github.com/Yoast/featurerequests/issues/422
